pyproject.toml

```diff
@@ -37,7 +37,7 @@ classifiers = [
 ]
 dependencies = [
   "awkward >=2.5.1",
-  "dask >=2023.04.0",
+  "dask >=2023.04.0,!=2025.4.0",
```

Comment:
Do we want a < in case another release comes out meanwhile?

Reply:
I can do that :+1: